packages/flutter/test/material/page_test.dart

```diff
@@ -23,46 +23,51 @@ void main() {
       ),
     );
 
-    final Offset widget1TopLeft = tester.getTopLeft(find.text('Page 1'));
-
     tester.state<NavigatorState>(find.byType(Navigator)).pushNamed('/next');
     await tester.pump();
+    await tester.pump(const Duration(milliseconds: 50));
+
+    ScaleTransition widget1Scale = tester.element(
+      find.text('Page 1'),
+    ).findAncestorWidgetOfExactType<ScaleTransition>();
+    ScaleTransition widget2Scale = tester.element(
+      find.text('Page 2'),
+    ).findAncestorWidgetOfExactType<ScaleTransition>();
+    FadeTransition widget2Opacity = tester.element(
+      find.text('Page 2'),
+    ).findAncestorWidgetOfExactType<FadeTransition>();
+
+    expect(widget1Scale.scale.value, greaterThan(1.0));
+    expect(widget2Scale.scale.value, lessThan(1.0));
+    expect(widget2Opacity.opacity.value, lessThan(1.0));
```

Comment:
I tried to migrate these tests, and it fails at here, with the actual value 1.0 .

```
The following TestFailure object was thrown running a test:
  Expected: a value less than <1.0>
  Actual: <1.0>
   Which: is not a value less than <1.0>
```

I'm wondering why it would be `1.0` after only 50 milliseconds, that doesn't seem correct.

Reply:
I'm not exactly sure why either and would have to look at your code. Do you have a branch with the attempted migration you can link me to?

Reply:
> Do you have a branch with the attempted migration you can link me to?

Not yet. Should I file a PR for this?

Reply:
The main issue I ran into originally was that it resulted in breaking changes internally at Google and I did not really know what the sources of the issue were. I would hold off on trying to get the Android 10 transition by default since it would require a Googler to fix the internal tests/implementations if we want to move forward.

Reply:
> I would hold off on trying to get the Android 10 transition by default since it would require a Googler to fix the internal tests/implementations if we want to move forward.

Because those tests/implementations are invisible to me, the only I can do is reach out to you.

If it re-opened at the future, please let me know. :)

Reply:
Or should we try again to see if it breaks them?

Reply:
I'm pretty sure it will continue to break the tests. I remember that they were mostly tests that rely strictly on the widgets contained in the existing default Android transition.